scripts/irods/test/test_prep_genquery_iterator.py

```diff
@@ -314,11 +331,13 @@ def main(rule_args,callback,rei):
             with generateRuleFile(names_list = self.to_unlink,
                                   prefix = file_prefix) as f:
                 rule_file = f.name
-                print(rule_text.format(**locals()), file=f, end='')
-            output, err, rc = self.admin.run_icommand("irule -F " + rule_file)
+                populated_rule_text = rule_text.format(**locals())
+                print(populated_rule_text) # For debugging.
+                print(populated_rule_text, file=f, end='')
+            output, err, rc = self.admin.run_icommand("irule -r irods_rule_engine_plugin-python-instance -F " + rule_file)
```

Comment:
Not sure if we have a preference on string vs list input to `run_icommand(...)`, just something to keep in mind if we do.

Reply:
While I do have a preference, it's not incorrect. Leaving as is due to time.